xcube/core/gen2/config.py

```diff
@@ -271,6 +277,14 @@ def get_schema(cls):
                     nullable=True,
                     pattern=r'^([1-9][0-9]*)?[DWMY]$'
                 ),
+                temporal_resampling=JsonStringSchema(
```

Comment:
See above.

Reply:
Closing to keep the conversation in one comment.